### ISSUE TITLE
Silverlight 4 binary

### ISSUE THREAD

**Issue by fholm** [fholm/IronJS#35]
IronJS needs to be able to run on SL4

**Comment by dotjosh** [comment]
This would be incredible (assuming MS doesn't end up killing off SL).  I may be forking shortly in a naive attempt to get something going.

**Comment by fholm** [comment]
@dotjosh: You're more then welcome, I think it's mostly an issue of duplicating functionality that is missing from the SL runtime but exists in the default CLR that IronJS relies on.

**Comment by Soul-Master** [comment]
Don't forget to create some example Silverlight project for implementing IronJS in milestone 0.2.1. Something like Sliverlight console for evaluate IronJS.

**Comment by fholm** [comment]
@Soul-Master: yes, we'll make sure to do that, first business is to get it running at all though :)
